### PR TITLE
Chore: lw7 - Day 1

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

Release Launch Week 7 - Day 1

## Additional context

Screenshot 2023-04-10 at 13 30 05

Screenshot 2023-04-10 at 13 30 12

Screenshot 2023-04-10 at 10 53 46

### CODE REVIEW

File: apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+title: Supabase Logs can now be self-hosted
```
Comment (kiwicopple) [inline apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx:2]: - Supabase Logs is now open source
- Self-hosted Logging Server 
- Supabase Logs: an open source logging server
- Supabase Logs: an open source Sentry alternative
- supabase logging is now integrated
- Supabase logs is now self-hostable
- Supabase Logs is integrated into the Supabase Stack

Comment (fsansalvadore) [reply apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx:2]: Let me know which one 😁

File: apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx [hunk 2]
```diff
@@ -0,0 +1,183 @@
+---
+title: Supabase Logs is now open source
```
Comment (kiwicopple) [inline apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx:2]:
```suggestion
title: Supabase Logs: open source logging server
```

Comment (Ziinc) [reply apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx]: “open source analytics server" maybe?

**Review by RamiroND — APPROVED** [review]

**Pull request merged**